filesystem/test_data/replicator_test/source/file1.txt

```diff
@@ -0,0 +1 @@
+This is file1
```

Comment:
@Prakhar-Agarwal-byte Please try creating a simple directory hierarchy in the test data and attempt the test.

Reply:
I changed it, is it fine now?